.gitignore

```diff
@@ -5,3 +5,5 @@ node_modules
 static/compiled
 static/fonts/font-awesome
 logs/
+.idea/
+/package-lock.json
```

Comment:
Remove `package-lock.json` from `.gitignore` and make sure it gets committed. It will help keep npm users up-to-date.